client/http-client/src/client.rs

```diff
@@ -225,7 +234,7 @@ impl ClientT for HttpClient {
 		.await
 	}
 
-	async fn batch_request<'a, R>(&self, batch: Vec<(&'a str, Option<ParamsSer<'a>>)>) -> Result<Vec<R>, Error>
+	async fn batch_request<R>(&self, batch: Vec<(&str, Option<Box<RawValue>>)>) -> Result<Vec<R>, Error>
```

Comment:
this looks like a more low-level API than before, can you explain why?

it's almost like we should provide two interfaces for this `one high-level` i.e. with generics and `one ow level` with RawValue as James implemented in subxt.

Reply:
I wanted to provide a way for users to construct the batch request as they wanted without any overhead, but the `BatchRequestBuilder` provides just that 😄